aspnetcore/fundamentals/aot.md

```diff
@@ -6,7 +6,7 @@ monikerRange: '>= aspnetcore-8.0'
 ms.author: midenn
 ms.custom: mvc
 ms.date: 04/10/2023
-uid: native-aot/index
+uid: fundamentals/aot
```

Comment:
Other docs in this area include native-aot in the URL - https://learn.microsoft.com/en-us/dotnet/core/deploying/native-aot/
```suggestion
uid: fundamentals/native-aot
```